helm-charts/splunk-otel-collector/templates/configmap-fluentd-cri.yaml

```diff
@@ -0,0 +1,49 @@
+{{ if and .Values.logsEnabled .Values.otelAgent.enabled }}
+{{/*
+Fluentd config parts applied only to clusters with conteinerd/cri-o runtime.
+*/}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "splunk-otel-collector.fullname" . }}-fluentd-cri
+  labels:
+    app: {{ template "splunk-otel-collector.name" . }}
+    chart: {{ template "splunk-otel-collector.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

Comment:
See this is used elsewhere but is there an adopted practice in helm/k8s to use "heritage" as opposed to "origin".  This is potentially a biased signifier (and also confusing imo).

Reply:
Thats for bringing this this! These are old-style labels used in helm 2. We should definately move from them, sumbitted an issue https://github.com/signalfx/splunk-otel-collector-chart/issues/155